keps/sig-node/1769-memory-manager/README.md

```diff
@@ -435,21 +501,19 @@ Memory pinning will be validated for Topology Manager `single-numa-node` and `re
 
 #### GA (stable)
 
-- User feedback
-- TBD
-
-<!--
-#### Removing a deprecated flag
+- N examples of real-world usage
```

Comment:
do we still want `N`? Do we have good examples to share? Maybe couple would be enough

Reply:
Agreed